ibm/service/kubernetes/resource_ibm_container_ingress_instance.go

```diff
@@ -0,0 +1,252 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package kubernetes
+
+import (
+	"fmt"
+	"time"
+
+	v2 "github.com/IBM-Cloud/bluemix-go/api/container/containerv2"
+	"github.com/IBM-Cloud/bluemix-go/bmxerror"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/validate"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func ResourceIBMContainerIngressInstance() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMContainerIngressInstanceCreate,
+		Read:     resourceIBMContainerIngressInstanceRead,
+		Update:   resourceIBMContainerIngressInstanceUpdate,
+		Delete:   resourceIBMContainerIngressInstanceDelete,
+		Exists:   resourceIBMContainerIngressInstanceExists,
+		Importer: &schema.ResourceImporter{},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"instance_crn": {
+				Type:        schema.TypeString,
```

Comment:
can we add ForceNew : true for instance_crn and cluster since it can't be updated

Reply:
done